### PR TITLE
Feature extraction config for BITS and Terminal Services

### PR DESCRIPTION
**Pull request by roshanmaskey**
Added Windows feature extraction configs for BITS client and Windows terminal services events.

**Checks**

- [x] All tests succeed.
- [ ] Unit tests added.
- [ ] e2e tests added.
- [ ] Documentation updated.

**Closing issues**

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes
(if such).

### CODE REVIEW

File: data/winevt_features.yaml [hunk 1]
```diff
@@ -648,3 +688,283 @@ system_7045_v0:
     - name: account_name
       string_index: 4
 
+# Microsoft-Windows-TerminalServices-LocalSessionManager/Operational
```
Comment (jkppr) [inline data/winevt_features.yaml:691]: Do you happen to have a reference for the `Microsoft-Windows-TerminalServices-LocalSessionManager` events?

File: data/winevt_features.yaml [hunk 2]
```diff
@@ -648,3 +688,283 @@ system_7045_v0:
...
+  event_identifier: 40
+  event_version: 0
+  references:
+  mapping:
+    - name: session_id
+      string_index: 0
+    - name: reason
+      string_index: 1
+
+# Microsoft-Windows-Bits-Client Events
```
Comment (jkppr) [inline data/winevt_features.yaml:788]: Can you add a reference for the `Microsoft-Windows-Bits-Client` events as well please?

**Review by jkppr — CHANGES_REQUESTED** [review]
Looking good, thanks a lot for the additions. For two sources there are references missing. It would be great if you could add them to the config file. Thanks

File: data/winevt_features.yaml [hunk 3]
```diff
@@ -648,3 +688,283 @@ system_7045_v0:
...
+  event_identifier: 3
+  event_version: 0
+  references:
+  mapping:
+    - name: job_title
+      string_index: 0
+    - name: job_owner
+      string_index: 1
+    - name: string3
+      string_index: 2
```
Comment (jkppr) [inline data/winevt_features.yaml:804]: @roshanmaskey where did you find a 3rd event field with BITS version 0? 
The documentation also lists only 2 fields: https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2008-r2-and-2008/cc734602(v=ws.10)

File: data/winevt_features.yaml [hunk 4]
```diff
@@ -648,3 +688,295 @@ system_7045_v0:
...
+
+### Version 2 is used in win2k16
+bits_client_3_v2:
+  source_name:
+    - Microsoft-Windows-Bits-Client
+  provider_identifier:
+    - '{EF1CC15B-46C1-414E-BB95-E76B077BD51E}'
+  event_identifier: 3
+  event_version: 2
+  references:
```
Comment (jkppr) [inline data/winevt_features.yaml:826]: @roshanmaskey Do you happen to have any reference for the mapping here? I was not able to find any documentation on the fields in version 1,2,3.

File: data/winevt_features.yaml [hunk 5]
```diff
@@ -648,3 +688,295 @@ system_7045_v0:
...
+### Version 1 is used in win2k16, win2k19, and win2k22
+bits_client_59_v1:
+  source_name:
+    - Microsoft-Windows-Bits-Client
+  provider_identifier:
+    - '{ef1cc15b-46c1-414e-bb95-e76b077bd51e}'
+  event_identifier: 59
+  event_version: 1
+  references:
+    - https://www.mandiant.com/resources/blog/attacker-use-of-windows-background-intelligent-transfer-service
```
Comment (jkppr) [inline data/winevt_features.yaml:902]: The mandiant blog does not contain any mention of event ID 59 nor the field mapping. Do you happen to have another reference?

File: data/winevt_features.yaml [hunk 6]
```diff
@@ -648,3 +688,295 @@ system_7045_v0:
...
+## Event ID 60: BITS stopped transferring the transfer job.
+bits_client_60_v1:
+  source_name:
+    - Microsoft-Windows-Bits-Client
+  provider_identifier:
+    - '{ef1cc15b-46c1-414e-bb95-e76b077bd51e}'
+  event_identifier: 60
+  event_version: 1
+  references:
+    - https://www.mandiant.com/resources/blog/attacker-use-of-windows-background-intelligent-transfer-service
```
Comment (jkppr) [inline data/winevt_features.yaml:938]: The mandiant blog does not contain any mention of event ID 60 nor the field mapping. Do you happen to have another reference?

**Review by jkppr — CHANGES_REQUESTED** [review]
I added some references that I was able to find. However, there are still some missing especially around BITS-Clients events. Please take a look below.

File: data/winevt_features.yaml (hunk 4 above)
Comment (roshanmaskey) [reply data/winevt_features.yaml:826]: I didn't find any documentation. These are based on the event logs created following Mandiant blog post.

File: data/winevt_features.yaml (hunk 3 above)
Comment (roshanmaskey) [reply data/winevt_features.yaml]: There should have been only two fields. Counting error at 0.

File: data/winevt_features.yaml (hunk 2 above)
Comment (roshanmaskey) [reply data/winevt_features.yaml:800]: There are mapping ref in this document https://community.microfocus.com/cfs-file/__key/communityserver-wikis-components-files/00-00-00-00-23/MSWinBITSClientEvtLogs.pdf.

File: data/winevt_features.yaml (hunk 4 above)
Comment (roshanmaskey) [reply data/winevt_features.yaml:826]: No external reference. These were based on events generated on different OSes. I have shared the plaso files with you.

File: data/winevt_features.yaml (hunk 5 above)
Comment (roshanmaskey) [reply data/winevt_features.yaml:902]: https://community.microfocus.com/cfs-file/__key/communityserver-wikis-components-files/00-00-00-00-23/MSWinBITSClientEvtLogs.pdf

File: data/winevt_features.yaml (hunk 6 above)
Comment (roshanmaskey) [reply data/winevt_features.yaml:938]: https://community.microfocus.com/cfs-file/__key/communityserver-wikis-components-files/00-00-00-00-23/MSWinBITSClientEvtLogs.pdf

**Pull request merged**